contrib/cirrus/cache_release.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+source $(dirname $0)/lib.sh
+
+req_env_var GOSRC CIRRUS_BUILD_ID
+
+cd $GOSRC
+
+make release_info.txt
+FIELDS="$RELEASE_BASENAME $RELEASE_VERSION $RELEASE_DIST $RELEASE_DIST_VER $RELEASE_ARCH $RELEASE_FILE"
+cat release_info.txt | read "$FIELDS"
```

Comment:
Again, this does not do what you think it do: `| read` is a subshell, so whatever gets read will be lost as soon as the pipe finishes.

As mentioned before, any time you see single-argument `cat` you should have alarm bells ringing in your head. In this case it should be:
```bash
read $FIELDS < release_info.txt
```

Reply:
Uggg, too many alarm bells, and they are always ringing :cry:   It's quieter to rely on observant reviewers...but I will attempt to notice single-headed cat's strolling by in scripts :smile:

Reply:
got it